application/src/main/java/org/opentripplanner/transit/service/TransitService.java

```diff
@@ -219,44 +210,45 @@ List<TripTimeOnDate> stopTimesForPatternAtStop(
     boolean includeCancellations
   );
 
-  Collection<GroupOfRoutes> getGroupsOfRoutes();
+  Collection<GroupOfRoutes> listGroupsOfRoutes();
 
-  Collection<Route> getRoutesForGroupOfRoutes(GroupOfRoutes groupOfRoutes);
+  Collection<Route> findGroupsOfRoutes(GroupOfRoutes groupOfRoutes);
 
-  GroupOfRoutes getGroupOfRoutesForId(FeedScopedId id);
+  @Nullable
+  GroupOfRoutes getGroupOfRoutes(FeedScopedId id);
 
   /**
    * Return the timetable for a given trip pattern and date, taking into account real-time updates.
    * If no real-times update are applied, fall back to scheduled data.
    */
   @Nullable
-  Timetable getTimetableForTripPattern(TripPattern tripPattern, LocalDate serviceDate);
+  Timetable findTimetable(TripPattern tripPattern, LocalDate serviceDate);
 
   /**
    * Return the real-time added pattern for a given tripId and a given service date.
    * Return null if the trip does not exist or if the trip has no real-time added pattern for
    * this date (that is: it is still using its scheduled trip pattern for this date).
    */
   @Nullable
-  TripPattern getNewTripPatternForModifiedTrip(FeedScopedId tripId, LocalDate serviceDate);
+  TripPattern findNewTripPatternForModifiedTrip(FeedScopedId tripId, LocalDate serviceDate);
 
   /**
    * Return true if at least one trip pattern has been modified by a real-time update.
    */
   boolean hasNewTripPatternsForModifiedTrips();
 
-  TripOnServiceDate getTripOnServiceDateForTripAndDay(TripIdAndServiceDate tripIdAndServiceDate);
+  TripOnServiceDate findTripOnServiceDate(TripIdAndServiceDate tripIdAndServiceDate);
 
   /**
    * Return the TripOnServiceDate for a given id, including real-time updates.
    */
-  TripOnServiceDate getTripOnServiceDateById(FeedScopedId datedServiceJourneyId);
+  TripOnServiceDate findTripOnServiceDate(FeedScopedId datedServiceJourneyId);
```

Comment:
This is again a bit on the edge, get or find?

Reply:
The argument name is old, it should be `tripOnServiceDateId`/`id`not `datedServiceJourneyId`. Again I prefear `id` because it is DRY.

Reply:
8a9b964cc7184440aef0c803a7b8f10a1f377af8